jobs/v3/src/main/java/com/google/samples/AutoCompleteSample.java

```diff
@@ -26,9 +26,9 @@
 /**
  * The samples in this file introduced how to do the auto complete, including:
  *
- * - Default auto complete (on both company display name and job title)
+ * <p>- Default auto complete (on both company display name and job title)
```

Comment:
You don't need markdown

Reply:
I used java-formatter, looks like formatter added some weird HTML tags. I will remove it